VAMobile/src/utils/hooks/index.tsx

```diff
@@ -46,6 +46,7 @@ import { capitalizeFirstLetter, stringToTitleCase } from 'utils/formattingUtils'
 import { isAndroid, isIOS, isIpad } from 'utils/platform'
 import { WaygateToggleType, waygateNativeAlert } from 'utils/waygateConfig'
 
+const textAlightment = isIOS() ? 'center' : 'left'
```

Comment:
Typo "Alightment". Could probably just call it "textAlign" and use shorthand for the object property